devtools/test_dashboard/test_gl3d.js

```diff
@@ -35,7 +35,9 @@ plots['marker-arrays'] = require('@mocks/gl3d_marker-arrays.json');
 plots['scatter3d-colorscale'] = require('@mocks/gl3d_scatter3d-colorscale.json');
 plots['autocolorscale'] = require('@mocks/gl3d_autocolorscale.json');
 plots['nan-holes'] = require('@mocks/gl3d_nan-holes.json');
+plots['connectgaps'] = require('@mocks/gl3d_scatter3d-connectgaps.json');
 plots['tet'] = require('@mocks/gl3d_tet.json');
 plots['surface_intensity'] = require('@mocks/gl3d_surface_intensity.json');
+plots['connectgaps'] = require('@mocks/gl3d_scatter3d-connectgaps.json');
```

Comment:
and this line should be:

```js
plots['scatter3d-connectgaps'] = require('@mocks/gl3d_scatter3d-connectgaps.json');
```

so that the corresponding [image baseline](https://github.com/plotly/plotly.js/blob/31c3b362d3fd3585de6f129af37dfb7cf0025424/devtools/test_dashboard/buttons.js#L29) is displayed properly.